mail-client/alpine/alpine-2.21.1.ebuild

```diff
@@ -4,9 +4,11 @@
 EAPI=6
 inherit autotools toolchain-funcs
```

Comment:
consider using `vcs-snapshot` over overriding `src_unpack`

Reply:
Fixed, thanks for the hint!